pypit/arutils.py

```diff
@@ -1076,6 +1076,61 @@ def subsample(frame):
     indices = coordinates.astype('i')
     return frame[tuple(indices)]
 
+def trace_gweight(fimage, xcen, ycen, sigma, invvar=None):
+    """ Port of SDSS trace_gweight algorithm
```

Comment:
Maybe it's best to provide just a little more information about the function, something along the lines of: `Determines the trace centroid by weighting the flux by an integral of a Gaussian over a pixel`. I'm not sure that this offers more clarity... but I think you know what I mean!!!